### ISSUE TITLE
Meson: Unknown variable "ctpp2_include_path"

### ISSUE THREAD

**Issue by orblivion** [kiwix/kiwix-lib#44]
I tried upgrading kiwix-lib from a3d01b6303c1ddf02a80d18b9283736c5e4f2f1c to 4485cc8d0f3a0aefddb5941ef65d49ef14ae2941 and I'm having a problem with Meson. I'm on 0.39.1, the latest available from pypi. I'm building on Debian Jessie. For reference, I had similar issues in #10.

```
The Meson build system
Version: 0.39.1
Source dir: /opt/app/kiwix-lib
Build dir: /opt/app/kiwix-lib/build
Build type: native build
Project name: kiwixlib
Native cpp compiler: c++ (gcc 4.9.2)
Build machine cpu family: x86_64
Build machine cpu: x86_64
Dependency threads found: YES
Found pkg-config: /usr/bin/pkg-config (0.28)
Native dependency icu-i18n found: YES 52.1
Native dependency libzim found: YES 1.4
Native dependency pugixml found: YES 1.8
Has header "ctpp2/CTPP2Logger.hpp": YES
Library ctpp2 found: YES
Program compile_resources.py found: YES (/opt/app/kiwix-lib/scripts/compile_resources.py)
Configuring kiwix_config.h using configuration

Meson encountered an error in file meson.build, line 87, column 5:
Unknown variable "ctpp2_include_path".
```

**Comment by kelson42** [comment]
I confirm the bug.

**Comment by mgautierfr** [comment]
fixed by @kelson42 with commit 9963c731
